pages/common/gcc.md

```diff
@@ -13,3 +13,7 @@
 - Include libraries from a different path:
 
 `gcc {{source.c}} -o {{executable}} -I{{header_path}} -L{{library_path}} -l{{library_name}}`
+
+- Export source code into Assembler instructions:
```

Comment:
I find "export" strange phrasing, or is it more common? I'd just use "compile to assembly"